### ISSUE TITLE
ProxyNCA refer to NCA Loss Formulation

### ISSUE THREAD

**Issue by alpoler** [KevinMusgrave/pytorch-metric-learning#206]
Hi Kevin, I read article of "No fuss distance metric learning". They give equation of NCA Loss different from original NCA loss paper. They include only anchor-negatives distance in denominator rather than all examples like in softmax.  This forms contradiction with hinton paper. You use formulation in hinton's paper. How do you conclude to utilize the original nca rather than the mentioned in proxyNCA loss ? @KevinMusgrave 

EDIT : I realized that first author also include positives in denominator and say that it does not deteriorate performance although he excludes positives theoretically in the paper.
